src/bindings/c/tests/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@
 #
 
 # OpenVINO Legacy C API test sample
+set(DEPENDENCIES mock_engine)
```

Comment:
I think it is not correct command. You need to use `add_dependencies()` command

Reply:
And do we still need to have dependencies on hw plugins below?

Reply:
> add_dependencies()
Will correct this.


> And do we still need to have dependencies on hw plugins below?
I think u are right, the hw plugins dependencies will be removed